science/gerris/Portfile

```diff
@@ -1,61 +1,74 @@
-PortSystem 1.0
+#-*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-name		gerris
-version		0.9.2
-categories	science
-license		GPL-2+
-platforms	darwin
-maintainers	nomaintainer
-description	The Gerris Flow Solver
-long_description	\
-		Gerris is an Open Source Free Software library		\
-		for the solution of the partial differential		\
-		equations describing fluid flow.
+PortSystem            1.0
+PortGroup             mpi 1.0
+PortGroup             compiler_blacklist_versions 1.0
 
-homepage	http://gfs.sourceforge.net/
-master_sites	sourceforge:gfs
+name		          gerris
+version		          131206
+categories	          science
+license		          GPL-2+
+platforms	          darwin
+maintainers           {mcalhoun @MarcusCalhoun-Lopez} openmaintainer
+description	          a program for solving fluid flow equations
+long_description	  Gerris is a Free Software program for the solution of the partial differential equations describing fluid flow.
+homepage	          http://gfs.sourceforge.net/
+distname              ${name}-snapshot-${version}
+master_sites          http://gerris.dalembert.upmc.fr/gerris/tarballs/
 
-checksums	md5 94e0fe2089052a3e6c689f85148388bc
+checksums             rmd160  5575376c8d9ce252215dd361443dfeb1c8e82144 \
+                      sha256  d5346a362b104ccc858c4b79938727d56c4654b103c268e54cf3aa56d0b55b39 \
+                      size    2841686
 
-depends_lib	path:lib/pkgconfig/glib-2.0.pc:glib2	\
-		port:gts
+# C99 required
+mpi.setup             require_fortran \
+                      -gcc44
 
-patchfiles	patch-Makefile.in
+# most of the patches are from Gentoo
+#    https://gitweb.gentoo.org/repo/gentoo.git/tree/sci-libs/gerris/gerris-20131206-r1.ebuild
+patchfiles-append     patch-fortran_lib.diff \
+                      patch-hypre_no_mpi.diff \
+                      patch-lis_names.diff
 
-use_parallel_build	no
+depends_build         port:pkgconfig
 
-post-destroot	{ xinstall -m 755 -d ${destroot}${prefix}/share/doc/${name}/examples
-		  xinstall -m 755 -d ${destroot}${prefix}/share/doc/${name}/examples/boussinesq
-		  xinstall -m 755 -d ${destroot}${prefix}/share/doc/${name}/examples/cylinder
-		  xinstall -m 755 -d ${destroot}${prefix}/share/doc/${name}/examples/cylinder/heated
-		  xinstall -m 755 -d ${destroot}${prefix}/share/doc/${name}/examples/rt
-		  xinstall -m 755 -d ${destroot}${prefix}/share/doc/${name}/examples/shear
+# N.B. wavewatch and agmg are non-free
+depends_lib	          path:lib/pkgconfig/glib-2.0.pc:glib2	\
+		              port:gts \
+                      port:proj \
+                      port:netcdf \
+                      port:hypre \
+                      port:gsl \
+                      port:fftw-3 \
+                      port:ode \
+                      port:lis
```

Comment:
Travis builds failed because there is no port called lis.

Reply:
This should now be fixed.
Sorry about not committing my local version.